code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +32,87 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
+	display_name = "a pack of medical Marijuana"
+	path = /obj/item/storage/fancy/cigarettes/cigpack_med
+
+/datum/gear/cigar
+	display_name = "Havana Cigar"
+	path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost = 2
+
+/datum/gear/smokingpipe
+	display_name = "smoking pipe"
+	path = /obj/item/clothing/mask/cigarette/pipe
+	cost = 2
+
 /datum/gear/lighter
 	display_name = "a cheap lighter"
 	path = /obj/item/lighter
 
+/datum/gear/matches
+	display_name = "a box of matches"
+	path = /obj/item/storage/box/matches
+
+/datum/gear/zippo
+	display_name = "zippo lighter"
+	path = /obj/item/lighter/zippo
+	cost = 2
+
+/datum/gear/candlebox
+	display_name = "a box candles"
+	description = "For setting the mood or for occult rituals."
+	path = /obj/item/storage/fancy/candle_box/eternal
+
 /datum/gear/rock
 	display_name = "a pet rock"
 	path = /obj/item/toy/pet_rock
 
+/datum/gear/redfoxplushie
```

Comment:
These are fun, but this is more and more encroaching on the purpose of the arcade.

Reply:
this is why I went with the base versions. If people want the fancy plushies they'll have to play the arcade but also mainly it's for people who want to have a friend that isn't a rock

Reply:
Still don't think a prize reward should be this easy to get, as that's a good portion of why you play the arcade, but not completely sure.

Reply:
I'm ok with this